setup.cfg

```diff
@@ -53,10 +53,9 @@ add-ignore =
     D105,
     D107,
     D202,
-    D203,
     D204,
-    D213,
     D413,
+    D416,
```

Comment:
what is this? couldn't find it here: http://www.pydocstyle.org/en/2.1.1/error_codes.html